web_generator/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
 ## 1.0.0-wip
 
-- Initial separation of `web_generator` from `web`.
\ No newline at end of file
+- Initial separation of `web_generator` from `web`.
+- New IDL interface `RHL` added. `ExtendedAttribute` idl interface updated to
```

Comment:
nit: I think it's okay to not include entries here, but this is okay too. This package is never published, so it's purely for internal purposes.

Reply:
I think it's nice to have this changelog updated with new updates. But I'm good either way. lmk if you want to remove this.

Reply:
Nope, that's okay - we can keep it. :) LGTM still!